lib/notify_slack/base.rb

```diff
@@ -0,0 +1,18 @@
+module NotifySlack
+  class Base
+    def initialize(student_id)
+      @notifier = Slack::Notifier.new(ENV['SLACK_WEBHOOK_URL'],
+                                      channel: ENV['SLACK_CHANNEL'],
+                                      username: ENV['SLAsCK_USERNAME'])
+      @student_id = student_id
+    end
+
+    attr_accessor :notifier, :student_id
+
+    def call
+      student = Courses::Student.find(student_id)
```

Comment:
Unfortunately coupling main app with Courses engine (and Students) is impossible.

Reply:
I mean, can we move this code into `courses` engine?